### ISSUE TITLE
View More functionality on recordset with faceting

### ISSUE THREAD

**Issue by jrchudy** [informatics-isi-edu/chaise#1310]
When `View More` is selected from a record page, we want consistent behavior for the filter and facet added in the recordset app with faceting.

We need to choose one of the two following when you select view more and navigate to recordset app:
 - The filter shows up above the recordset table and the facet appears in the left panel. When clear all is selected, the filter disappears and the facet panel is unchanged. The facet that was added remains in the UI (even though it's not part of the annotation or heuristics or URI)

OR
 - The filter shows up above the recordset table and the facet is *NOT* shown in the left panel. When clear all is selected, the filter disappears and the facet panel is unchanged

Basically, we need to consistently always show the facet if we added it to the UI even after there are no filters for it or never show the facet (when not part of the annotation, heuristics, or URI)

**Comment by jrchudy** [comment]
Another thing to think about with this issue is what to do in the case of `showFaceting: false`? We discussed not showing the facet panel at all (which is the behavior of `showFaceting: false`) but still showing the filters as if the data is being filtered by a facet.
